src/main/java/me/juancarloscp52/bedrockify/mixin/client/features/panoramaBackground/EntryListWidgetMixin.java

```diff
@@ -34,21 +36,19 @@ private void renderPanorama(EntryListWidget entryListWidget, MatrixStack matrice
         }
     }
 
-    // Prevent the screen background from drawing
-    @Redirect(method = "render", at=@At(value = "FIELD",target = "Lnet/minecraft/client/gui/widget/EntryListWidget;renderBackground:Z", opcode = Opcodes.GETFIELD))
-    private boolean doNotDrawBackground(EntryListWidget entryListWidget){
-        return !Bedrockify.getInstance().settings.isCubemapBackgroundEnabled() || shouldIgnoreScreen();
+    @Inject(method = "<init>(Lnet/minecraft/client/MinecraftClient;IIIII)V", at=@At("TAIL"))
```

Comment:
```suggestion
    @Inject(method = "<init>(Lnet/minecraft/client/MinecraftClient;IIIII)V", at=@At("RETURN"))
```

Should we use `RETURN` in At annotation?

https://fabricmc.net/wiki/tutorial:mixin_injects says

> **RETURN** Before *every* return statement
> **TAIL** Before *the final* return statement

It might be more compatible with other MOD that use `cancelable=true` and `CallbackInfo#cancel`.